### PR TITLE
v4.9 Release

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-44053
Staging - https://deploy-preview-110--docs-pymongo.netlify.app/whats-new/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -15,15 +15,27 @@ What's New
...
 
+.. _version-4.9:
+
+What's New in 4.9
+-----------------
+
+- Adds support for MongoDB 8.0 and Python 3.13.
+- Adds support for range queries with Queryable Encryption.
+- Adds a new asynchronous API. This API is in beta and is subject to change
+  before full release.
```
Comment (rustagir) [inline source/whats-new.txt:36]:
```suggestion
- Adds support for {+mdb-server+} 8.0 and Python 3.13.

- Adds support for Queryable Encryption (QE) range queries. To use this
  feature, your app must connect to {+mdb-server+} 8.0 or later. For
  more information about QE range queries, see :manual:`Queryable
  Encryption </core/queryable-encryption>` in the {+mdb-server+} manual.
  
- Adds a new asynchronous API. This API is in beta and is subject to change
  before the generally available release.
```

File: source/whats-new.txt [hunk 2]
```diff
@@ -15,15 +15,27 @@ What's New
...
+.. _version-4.9:
+
+What's New in 4.9
+-----------------
+
+- Adds support for MongoDB 8.0 and Python 3.13.
+- Adds support for range queries with Queryable Encryption.
+- Adds a new asynchronous API. This API is in beta and is subject to change
+  before full release. 
+.. TODO: Add link to the Pymongo Async migration guide
```
Comment (rustagir) [inline source/whats-new.txt:37]: S: While this is a TODO, could you add more info about what this means for motor to the whats new item?

File: source/includes/mongodb-compatibility-table-pymongo.rst [hunk 3]
```diff
@@ -21,7 +21,7 @@
      - ✓
      - ✓
      - ✓
-     - ✓
+     -
```
Comment (rustagir) [inline source/includes/mongodb-compatibility-table-pymongo.rst:24]: Are there language compatibility changes needed too?

**Review by rustagir — APPROVED** [review]
lgtm, non blocking comments but can take another look if needed!

File: source/includes/mongodb-compatibility-table-pymongo.rst (hunk 3 above)
Comment (jordan-smith721) [reply source/includes/mongodb-compatibility-table-pymongo.rst:24]: No, the language compat changes were already in master for this version

**Pull request merged**